{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "158ea802_b35e3d92",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1000120
      },
      "writtenOn": "2022-11-24T07:54:09Z",
      "side": 1,
      "message": "Is the CI error unrelated to this patch?",
      "revId": "db2bf3ac193f66f365b962b911e7bb2ffbde0a25",
      "serverId": "8f6f209b-db1a-4cbf-aa44-c8bc30e9bfda"
    }
  ]
}